cmd/clusterctl/client/cluster/cert_manager.go

```diff
@@ -220,86 +218,79 @@ func (cm *certManagerClient) createObj(o unstructured.Unstructured) error {
 	return nil
 }
 
-// getWebhook returns the cert-manager Webhook or nil if it does not exists.
-func (cm *certManagerClient) getWebhook() (*unstructured.Unstructured, error) {
+// allObjectsExist checks if all the objects with the given name already exist
+func (cm *certManagerClient) allObjectsExist(ctx context.Context, log logr.Logger, objs []unstructured.Unstructured) (bool, error) {
```

Comment:
nit
```suggestion
func (cm *certManagerClient) allObjectsExist(ctx context.Context, objs []unstructured.Unstructured) (bool, error) {
```
We usually avoid to pass the logger around and instead use `log := logf.Log` to access to the global logger; same apply to other func